client/src/Movies/SavedList.js

```diff
@@ -10,9 +11,11 @@ export default class SavedList extends Component {
       <div className="saved-list">
         <h3>Saved Movies:</h3>
         {this.props.list.map(movie => (
-          <span className="saved-movie">{movie.title}</span>
+          <NavLink to={`/movies/${movie.id}`} className="saved-movie">{movie.title}</NavLink>
         ))}
-        <div className="home-button">Home</div>
+        <Link to="/">
+          <div className="home-button">Home</div>
+        </Link>
       </div>
```

Comment:
You could also use a `NavLink` for the home button so that you can indicate to the user if they are currently in the home page.